src/Orleans/AssemblyLoader/AssemblyProcessor.cs

```diff
@@ -75,7 +75,20 @@ public void Initialize()
                 Assembly[] assemblies = AppDomain.CurrentDomain.GetAssemblies();
 
                 // initialize serialization for already loaded assemblies.
-                CodeGeneratorManager.GenerateAndCacheCodeForAllAssemblies();
+                var generated = CodeGeneratorManager.GenerateAndCacheCodeForAllAssemblies();
+                if (generated != null)
+                {
+                    foreach (var generatedAssembly in generated)
+                    {
+                        this.ProcessAssembly(generatedAssembly?.Assembly);
+                    }
+                }
+
+                foreach (var generatedAssembly in CodeGeneratorManager.GetGeneratedAssemblies().Values)
+                {
+                    this.ProcessAssembly(generatedAssembly?.Assembly);
+                }
+
```

Comment:
in net462 we rely on assembly load events to process generated assemblies. In netstandard1.5, no such event exists, so we pass generated assemblies back from codegen and process them immediately.